### ISSUE TITLE
Docs: Clarify debugging flags

### ISSUE THREAD

**Issue by swashcap** [electron/electron#7756]
- Electron version: 1.4.4
- Operating system: OS X 10.21.1

The [“Debugging the Main Process” doc page](http://electron.atom.io/docs/tutorial/debugging-main-process/) explains how to use the command line switches. However, when running:

``` shell
$ node_modules/.bin/electron . --debug=5859

  error: unknown option `--debug'
```

The proper way to use the flag:

``` shell
$ node_modules/.bin/electron --debug=5859 .
# main process runs...
```

The docs should be updated, possibly with an example of how to use Node.js’s built-in debugger.

**Comment by julianengel** [comment]
I attempted to do this, hope this is what was asked for :)

**Comment by kevinsawicki** [comment]
Fixed by #7787

**Comment by swashcap** [comment]
:+1: Nice!
